paper.tex

```diff
@@ -291,17 +291,43 @@ \subsubsection{A Scalable Keys and Data Entry Points Management}
 
 \subsection{Delegate MAM operations to Tangle-accelerator}
 \label{section:ta_endpoint}
-MAM builds a secure and authenticated communication protocol on top of multiple cyptosystems, which low-level devices need to spend a lot of resources to apply it, and they may not even support built-in libraries in regular operating system. To overcome these concerns, we propose another approach that allows data providers to delegate the MAM operations to Tangle-accelerators, proxy servers with high computation power, which can accelerates the interactions with Tangle while ensuring the privacy of data providers.
+MAM builds a secure and authenticated communication protocol on top of multiple cryptosystems, which low-level devices need to spend a lot of resources to apply it, and they may not even support built-in libraries in regular operating system. To overcome these concerns, we propose another approach that allows data providers to delegate the MAM operations to Tangle-accelerators, proxy servers with high computation power, which can accelerates the interactions with Tangle while ensuring the privacy of data providers.
 
-\subsubsection{Plight for Low-level edge device}
+\subsubsection{Plight for Low-level Edge Device}
+MAM requires quite huge computational power which causes a burden for low-level devices to adopt MAM service. Therefore we introduce a proxy server, tangle-accelerator, which performs the heavy MAM computations, so low-level devices can be relieved from the enormous computation.
+
+\subsubsection{Infrastructure}
+To build the system, MQTT is chosen as the communication protocol. MQTT protocol utilizes much more lightweight packet than HTTP protocol has. The topic-based publish-subscribe paradigm and lightweight packet that MQTT adopts would undoubtedly play an important role on low-level edge devices with limited bandwidth. The publish-subscribe model decouples publishers and subscribers in many ways.
+
+\begin{itemize}
+    \item Space decoupling: Space decoupling allows subscriber and publisher don’t need to know the information of each other
+    \item Time decoupling: Publishers and subscribers do not need to run at the same time
+    \item Synchronization decoupling: Operations are decoupled from the publisher and subscriber. If we can decouple the operations, then the endpoint doesn’t need to maintain the connection until receive the response. Message from endpoint can be served as send-and-forget fashion.
+\end{itemize}
+
+\subsubsection{End-to-End-Encryption}
+Blockchain operations are known for high computational power consuming. Such a characteristics makes a burden for including low-level embedded system to blockchain service. Therefore, we introduce a proxy server which would process the blockchain operations for the low-level embedded system. However, introducing a proxy server to process all the cryptographic operations causes the proxy server or the middle man can easily tamper the message.
```

Comment:
The sentences before `However,` is similar to the first paragraph.
Remove and we can move on to the concerns directly.